wallet/src/Pos/Wallet/Web/ClientTypes/Types.hs

```diff
@@ -269,7 +269,7 @@ instance Default CAccountMeta where
 -- | Query data for wallet creation
 data CWalletInit = CWalletInit
     { cwInitMeta     :: !CWalletMeta
-    , cwBackupPhrase :: !BackupPhrase
+    , cwBackupPhrase :: !Mnemonic
```

Comment:
same as in L299 - I believe this is a breaking change for V0 users as json serilization format changed.

Reply:
See above comment, this shouldn't matter in the end.